src/System.Transactions/src/System/Transactions/TransactionTable.cs

```diff
@@ -0,0 +1,675 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I know that this comment doesn't belong here, but the browser won't show me TransactionState.cs because it is too big.

In the header comments for TransactionStatePSPEOperation in lines 5015/5016, it says "Don't allow them to do anything stupid." Please change this to "Don't allow anything that is not supported."